bin/_test-run.sh

```diff
@@ -136,16 +136,18 @@ check_if_k8s_reachable(){
     printf '[ok]\n'
 }
 
-remove_l5d_if_exists() {
-  resources=$(kubectl --context=$k8s_context get all,clusterrole,clusterrolebinding,mutatingwebhookconfigurations,validatingwebhookconfigurations,psp,crd -l linkerd.io/control-plane-ns --all-namespaces -oname)
-  if [ ! -z "$resources" ]; then
-    printf 'Removing existing l5d installation...'
-    cleanup
+check_if_l5d_exists() {
+    printf 'Checking if Linkerd resources exist on cluster...'
+    resources=$(kubectl --context=$k8s_context get all,clusterrole,clusterrolebinding,mutatingwebhookconfigurations,validatingwebhookconfigurations,psp,crd -l linkerd.io/control-plane-ns --all-namespaces -oname)
+    if [ ! -z "$resources" ]; then
+        printf '\nLinkerd resources exist on cluster:\n\n'
+        printf '%s\n' "$resources"
+        printf '\nHelp:\n'
+        printf "    Run: [%s/test-cleanup]\n" "$bindir"
+        printf "    Specify a cluster context: [%s/test-run %s [%s] [context]]\n" "$bindir" "$linkerd_path" "$linkerd_namespace"
```

Comment:
It is more resource efficient to keep this down to one `printf` call (and then split the line in the script for maintenance readability) instead of multiple ones.

Reply:
Thanks! This is now one `printf`